Here is the error:

In file included from third_party/tensorflow/core/kernels/data/tf_record_dataset_op_test.cc:14:
./third_party/tensorflow/core/kernels/data/dataset_test_base.h:394:23: error: field 'compare_order' is uninitialized when used here [-Werror,-Wuninitialized]
        compare_order(compare_order) {}
                      ^
third_party/tensorflow/core/kernels/data/tf_record_dataset_op_test.cc:158:11: note: in instantiation of member function 'tensorflow::data::SkipTestCase<tensorflow::data::(anonymous namespace)::TFRecordDatasetParams>::SkipTestCase' requested here
  return {{/*dataset_params=*/TFRecordDatasetParams1(),

--- a/tensorflow/core/kernels/data/dataset_test_base.cc
+++ b/tensorflow/core/kernels/data/dataset_test_base.cc
@@ -616,7 +616,8 @@ Status DatasetOpsTestBase::CheckIteratorSkip(
 
   bool end_of_sequence = false;
   int num_skipped = 0;
-  iterator->Skip(ctx, num_to_skip, &end_of_sequence, &num_skipped);
+  TF_RETURN_IF_ERROR(
+      iterator->Skip(ctx, num_to_skip, &end_of_sequence, &num_skipped));
   EXPECT_TRUE(num_skipped == expected_num_skipped);
   if (get_next) {
     EXPECT_TRUE(!end_of_sequence);

--- a/tensorflow/core/kernels/data/dataset_test_base.h
+++ b/tensorflow/core/kernels/data/dataset_test_base.h
@@ -386,7 +386,7 @@ struct SkipTestCase {
   SkipTestCase(T dataset_params, int num_to_skip,
                int expected_num_skipped, bool get_next = false,
                std::vector<Tensor> expected_outputs = {},
-               bool compare_ordered = true)
+               bool compare_order = true)
       : dataset_params(std::move(dataset_params)),
         num_to_skip(num_to_skip),
         expected_num_skipped(expected_num_skipped),

